### ISSUE TITLE
Some users of urllib3 need it pinned to "<2"

### ISSUE THREAD

**Issue by mdmintz** [seleniumbase/SeleniumBase#1931]
For some users of ``Python<3.10``, setting ``urllib3`` to the latest major version (``>=2.0.0``) can lead to errors, (such as ``ImportError: urllib3 v2.0 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with OpenSSL 1.0.2k-fips 26 Jan 2017. See: https://github.com/urllib3/urllib3/issues/2168``). This is discussed in https://github.com/urllib3/urllib3/issues/2168

Currently, ``seleniumbase`` has this range in the ``setup.py`` and ``requirements.txt`` files:

``urllib3>=1.26.16,<2.1.0;python_version>="3.7``

That's not a problem if people pin ``urllib3==1.26.16`` themselves, but perhaps it's best that we pin the version for them to save them time, (for ``Python<3.10``). Although not all users of ``Python<3.10`` will encounter this issue if no pin is done, it probably won't hurt to pin it there anyway. Python users ``>3.10`` should be fine with ``urllib3>=1.26.16,<2.1.0`` for the larger range.

**Comment by mdmintz** [comment]
I'm considering using this range for the next release:

```python
urllib3==1.26.12;python_version<"3.7"
urllib3>=1.26.16,<2;python_version>="3.7" and python_version<"3.10"
urllib3>=1.26.16,<2.1.0;python_version>="3.10"
```

**Comment by mdmintz** [comment]
Resolved in ``4.15.7`` - https://github.com/seleniumbase/SeleniumBase/releases/tag/v4.15.7
